notebooks/118-optimize-preprocessing/118-optimize-preprocessing.ipynb

```diff
@@ -367,15 +373,15 @@
    "source": [
     "### Preprocessing Steps\n",
     "\n",
-    "Now, the sequence of preprocessing steps can be defined. More information about preprocessing steps, please, see [here](https://docs.openvino.ai/latest/classov_1_1preprocess_1_1PreProcessSteps.html#class-ov-preprocess-preprocesssteps).\n",
+    "Now, the sequence of preprocessing steps can be defined. For more information about preprocessing steps, see the [PreProcessSteps](https://docs.openvino.ai/latest/classov_1_1preprocess_1_1PreProcessSteps.html#class-ov-preprocess-preprocesssteps).\n",
```

Comment:
not sure that it is good idea to use camel case for link name, as there is no direct usage this class in the code below, it is not obvious that it is class name and how it is connected with common flow. Also python API link will be more suitable than c++
https://docs.openvino.ai/latest/api/ie_python_api/_autosummary/openvino.preprocess.PreProcessSteps.html#openvino.preprocess.PreProcessSteps